src/shaders/symbol_sdf.fragment.glsl

```diff
@@ -1,12 +1,19 @@
+#define SDF_PX 8.0
+#define BLUR_OFFSET 1.19
+#define HALO_OFFSET 6.0
+#define GAMMA 0.105/DEVICE_PIXEL_RATIO
```

Comment:
# Opinion Below 👇 

I find some of these constant names cryptic:

 - What's the difference between `GAMMA` and `gamma`?
 - What's meant by `SDF_PX`? SDF pixels? 
 - Why is `halp_blur` _multiplied_ by `BLUR_OFFSET`? 
 - What's a `buff`? Is it a buffer? 

I prefer a very clear constant name (ideally googleable) to a magic number, and a magic number to a cryptic constant name.